rules/60-sensor.rules

```diff
@@ -7,4 +7,8 @@ SUBSYSTEM=="iio", KERNEL=="iio*", SUBSYSTEMS=="usb|i2c", \
   IMPORT{builtin}="hwdb 'sensor:modalias:$attr{modalias}:$attr{[dmi/id]modalias}'", \
   GOTO="sensor_end"
 
+DRIVERS=="cmpc_accel_v4", \
```

Comment:
@hadess this rule works with no iio-sensor-proxy changes applied:
```
SUBSYSTEM=="input", ENV{ID_INPUT_ACCELEROMETER}=="1", SUBSYSTEMS=="usb|i2c|acpi", \
  IMPORT{builtin}="hwdb 'sensor:modalias:$attr{modalias}:$attr{[dmi/id]modalias}'", \
  GOTO="sensor_end"
```

It matches both input8 and event7.

When processing input8, input8 has a modalias `input:b0000v0000p0000e0000-e0,3,kra0,1,2,mlsfw` which is used for $attr{modalias} so that is looked up in hwdb with no match, so no matrix applied.

When processing event7, there is no modalias so it falls back to the matched acpi parent and uses that modalias, hwdb lookup succeeds and it applies the matrix.

Does this sound like a good approach?